docs/serving/creating-domain-mappings.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
-linkTitle: "Creating Domain Mappings (Alpha)"
+title: "Setting up a custom domain per Service"
+linkTitle: "Setting up a custom domain per Service"
 weight: 64
 type: "docs"
+aliases:
+  - /docs/serving/using-a-custom-domain-per-service
```

Comment:
You want to add an alias for the file you're deleting, to redirect it here:

```suggestion
  - /docs/serving/using-a-custom-domain-per-service
  - /docs/serving/services/using-a-custom-domain-per-service
```

Reply:
@evankanderson thanks. Committed suggestion.